frontend/src/reducer/searchResult.js

```diff
@@ -16,16 +45,31 @@ const defaultResult = {
 }
 
 function uniqueResults (allResults) {
-  return uniqBy(allResults, 'content_id')
+  return uniqBy(allResults, 'contentId')
```

Comment:
I know this function isn't new in this PR but it isn't named well.
This name is too generic, it feels that we can call it on any array and it'll guaranty unicity on it. But internally, it only filters on "content_id"

You have to name utility function from what they are doing and not from their context of use.
Here, it will filter the search results so the name uniqueResults make sense but if we look at it's behavior, all it does is filtering an array on the property contentId.
So I would call it uniqueArrayByContentId, or uniqueByContentId (listToFilter)

Reply:
alternatively, while, in the reducers, we usually do everything in the return statement, it is only because it usually is easy enough to do so.
But you can write multiple statements to have something like this:
``` javascript
const newResultList =  action.newSearchResultsList.map(item => serialize(item, serializeSearchItemProps))
const uniqueResultList = uniqBy(newResultList, 'contentId')
return { ...state, resultsList: uniqueResultList }
```
instead of:
``` javascript
return {
  ...state,
  resultsList: uniqueResults((action.newSearchResultsList).map(item => serialize(item, serializeSearchItemProps)))
}
```
In this particular case, it is even one line shorter